types/staking.go

```diff
@@ -22,15 +22,15 @@ const (
 	ValidatorUpdateDelay int64 = 1
 )
 
-// PowerReduction is the amount of staking tokens required for 1 unit of consensus-engine power
-var PowerReduction = NewIntFromBigInt(new(big.Int).Exp(big.NewInt(10), big.NewInt(6), nil))
+// DefaultPowerReduction is the default amount of staking tokens required for 1 unit of consensus-engine power
+var DefaultPowerReduction = NewIntFromBigInt(new(big.Int).Exp(big.NewInt(10), big.NewInt(6), nil))
```

Comment:
Why isn't this global variable just removed entirely? It'd make sense to me if it existed in just testing files

Reply:
We need to have a default available, in order to generate the params for a new genesis file.

See:  https://github.com/cosmos/cosmos-sdk/blob/master/x/staking/types/params.go#L73

The `sdk.DefaultBondDenom` is also available in a similar way.

Reply:
I see. Its unfortunate this isn't architected to just be a single default params struct per module